serde-generate/tests/typescript_generation.rs

```diff
@@ -46,9 +46,8 @@ fn write_package_tsconfig_json_for_test_build(path: std::path::PathBuf) -> Resul
     "skipLibCheck": true,
     "forceConsistentCasingInFileNames": true,
     "lib": [
-      "es6",
-      "esnext.BigInt",
-      "dom"
+//      "es6",
```

Comment:
In L40 above, we target es6. With it no longer being esnext, perhaps BigInt is not in the standard without the need for libs?

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects#numbers_and_dates